app/coffee/ConnectedUsersManager.coffee

```diff
@@ -42,7 +43,7 @@ module.exports =
 	refreshClient: (project_id, client_id, callback = (err) ->) ->
 		logger.log project_id:project_id, client_id:client_id, "refreshing connected client"
 		multi = rclient.multi()
-		multi.hset Keys.connectedUser({project_id, client_id}), "last_updated_at", Date.now()
+		multi.hset Keys.connectedUser({project_id, client_id}), "socket_checked_at", Date.now()
```

Comment:
You could expire the `last_response_at` key directly after `REFRESH_TIMEOUT_IN_S` and save some processing overhead.

Namely the date creation here, and most of the parsing and date logic below.

`_isInactive` is then as simple as `return connected_user.socket_checked_at? `.

I would rename the key once again to `is_active`.

I was about to suggest this in the last PR but refused to due to the storage overhead in redis.